packages/nameguard-react/src/components/NameShield/index.tsx

```diff
@@ -1,50 +1,46 @@
 import React, { type ReactNode } from "react";
-import { ConsolidatedNameGuardReport, Rating } from "@namehash/nameguard";
+import { ConsolidatedNameGuardReport } from "@namehash/nameguard";
 import cc from "classcat";
 
 import { Tooltip } from "../Tooltip/Tooltip";
-import { Shield, ShieldSize } from "../Report/Shield";
-
-function textColor(rating?: Rating) {
-  switch (rating) {
-    case Rating.alert: {
-      return "text-red-600";
-    }
-    case Rating.pass: {
-      return "text-emerald-600";
-    }
-    case Rating.warn: {
-      return "text-amber-500";
-    }
-    default: {
-      return "text-gray-500";
-    }
-  }
-}
+import {
+  Shield,
+  ShieldSize,
+  getNameGuardRatingTextColors,
+} from "../Report/Shield";
+import { LoadingShield } from "../icons/LoadingShield";
+import { ErrorShield } from "../icons/ErrorShield";
 
 type NameShieldProps = {
   data?: ConsolidatedNameGuardReport;
   children?: ReactNode;
   disableHover?: boolean;
   size?: ShieldSize;
-} & React.ComponentProps<typeof Shield>;
+  error?: string;
+} & React.ComponentProps;
 
 export function NameShield({
```

Comment:
Great that you've raised this topic. It seems to me that the ReportShield (old Report/Shield.tsx) is not previewed in NameGuard figma specs. This is the difference between ReportShield and NameShield. I suggest we remove ReportShield and update NameShield name to ReportShield. @notrab any ideas around the shared infos above?

Reply:
Uploading Gravação de Tela 2024-04-29 às 13.48.58.mov…

Reply:
I don't have the full context of the discussion about this but at the time of creating these, there were components which are specific to our website report, and Nameguard library itself, which is probably where some of the confusion is.

These components were created in the space of a week, 4 months ago. So there's likely a better way to do it now, but keep in mind that the website and report required different visuals for the same rating.

Reply:
Keeping this in mind I went to review Website's implementation and there is no usage to the what was known as ReportShield, currently, thus, I have deleted this file and named the current NameShield as ReportShield, which is a more aligned name for this component since it presents a ShieldIcon + Report details in a tooltip.